### PR TITLE
Add a basic Gooey character from GooKeeper

### PR DESCRIPTION
**Pull request by AndyTechGuy**
Adds a second basic example character, taken and adapted from the GooKeeper module. A green Gooey that operates with the 'stray' behavior.

### CODE REVIEW

File: assets/prefabs/strayGooey.prefab [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+      0,
+      255
+    ]
+  },
+  "skeletalmesh": {
+    "mesh": "gooey",
+    "heightOffset": -2.0,
+    "material": "gooey",
+    "animationPool": [
+      "gooeyIdleBreathing",
```
Comment (Cervator) [inline assets/prefabs/strayGooey.prefab]: This makes me think we should include block drop grammar type extras in these pools, so we could just write `gooeyIdleBreathing*21` ... 😁 

Not a change for this PR, just thinking out loud

File: assets/prefabs/strayGooey.prefab [hunk 2]
```diff
@@ -0,0 +1,117 @@
+{
+  "alwaysRelevant": true,
```
Comment (Cervator) [inline assets/prefabs/strayGooey.prefab]: I dunno if 2 spaces or 4 spaces in JSON is the more common one and I'm not in a hurry to figure it out and fix all files everywhere, but you might want to go for just one in the same PR :D

**Review by Cervator — APPROVED** [review]
This is fine and works, although you could standardize the indentation just a bit if you like. But while it is cool to see a green gooey just wandering around it is also kinda boring since the deer already does that. To shine up this PR a bit how about finding something unique for this Gooey to do that illustrates something new and useful for the tutorial? :-)

You could also put some updated and related info in the wiki, although right now the one from here actually lives in the base Behavior module. Maybe you could expand a bit on https://github.com/Terasology/Behaviors/wiki/Examples-and-TutorialBehaviors which looks like it never really got filled out?

**Comment by AndyTechGuy** [comment]
I was going to make a different PR expanding on the basic gooey character, but  I'll do it here if it would be a better place for it :-)

I'm planning on making gooeys that exhibit different behaviors marked by colour (again like GooKeeper), such as following/running away/attacking the player, and other gooeys that exhibit wildly different behaviors not in GooKeeper or in other behavior modules. For example a gooey that wants to stay in the dark, or a gooey that wants to find water to stay in, etc.

**Review by skaldarnar — APPROVED** [review]

**Pull request merged**